prow/cluster/jobs/istio/tools/istio.tools.yaml

```diff
@@ -0,0 +1,74 @@
+presubmits:
+
+  istio/tools:
+
+    - name: tools-lint
+      path_alias: istio.io/tools
+      decorate: true
+      always_run: true
+      optional: true
+      spec:
+        containers:
+          - image: gcr.io/istio-testing/circleci:2019-06-13
```

Comment:
not sure what is in this image.  since optinoal no real harm done if this pr merged.